examples/react-router/app/routes/json.tsx

```
@@ -0,0 +1,126 @@
+					<div>{fields.firstName.errors}</div>
+				</div>
+				<div>
+					<label>Media</label>
+					<input
+						type="text"
+						defaultValue={fields.media.defaultJSON}
+						name={fields.media.nameForJSON}
+					/>
+					<div>{fields.media.errors}</div>
+				</div>
+				<button disabled={!dirty}>Save</button>
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)